app/components/marketplace/stripe_overview_component.rb

```diff
@@ -7,6 +7,5 @@ def initialize(marketplace:, **kwargs)
     super(**kwargs)
 
     @marketplace = marketplace
-    @marketplace_stripe_utility = marketplace.stripe_utility
```

Comment:
This was raising the `RecordNotFound` on instantiation; which was causing the 404!